README.md

```diff
@@ -48,7 +48,11 @@ asdf install
 Make sure you have the following services configured and running on your development background:
 
 * [PostgreSQL](https://www.postgresql.org)
-* [Redis](https://redis.io)
+* [Redis]*(https://redis.io)
+
+### *Redis
+You can check if Redis has been installed by - `redis-cli -v`
```

Comment:
This only checks the client is installed. We should start the server and test the connection. On mac:
```
% redis-server
5227:C 24 Mar 2021 11:01:45.921 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
...
% redis-cli                                                                                                                                                                                                                         11:02:11
127.0.0.1:6379> 
```